game/CMakeLists.txt

```
@@ -108,55 +108,90 @@ foreach(lib ${OUR_LIBS} SDL2 PangoCairo LibRSVG LibXML++ AVFormat SWResample SWS
 	add_definitions(${${lib}_DEFINITIONS})
 endforeach(lib)
 
-option(ENABLE_MIDI "Enable support for MIDI hardware (e-drums with USB/MIDI connection)." ON)
-if(ENABLE_MIDI)
-	find_package(PortMidi)
+# Activating MIDI hardware (e-drums with USB/MIDI connection)
+set(ENABLE_MIDI AUTO CACHE STRING "Use midi hardware (e-drums with USB/MIDI connection)")
+set_property(CACHE ENABLE_MIDI PROPERTY STRINGS AUTO ON OFF)
+if(NOT DEFINED ENABLE_MIDI OR "AUTO" STREQUAL ${ENABLE_MIDI})
+	# Midi support not requested explicitly, best effort try to find it
+	find_package(PortMidi QUIET)
 	if(PortMidi_FOUND)
-		include_directories(${PortMidi_INCLUDE_DIRS})
-		list(APPEND LIBS ${PortMidi_LIBRARIES})
-		add_definitions("-DUSE_PORTMIDI")
-		message(STATUS "MIDI I/O: Enabled")
+		message(STATUS "MIDI I/O support: Enabled (automatically found)")
 	else()
-		message(STATUS "MIDI I/O: Disabled (libportmidi not found)")
+		message(STATUS "MIDI I/O support: Disabled (libportmidi not found)")
 	endif()
+elseif(ENABLE_MIDI)
```

```suggestion
elseif("${ENABLE_MIDI}" STREQUAL "ON")
```
I guess you cannot disable it if you keep the 'if' as is

---

it works nicely with on, I think I prefer the default ON/OFF without quoting

---

ok